### PR TITLE
Release v0.8.0

### PR DESCRIPTION
**Pull request by tuofeilunhifi**

### CODE REVIEW

File: docs/source/change_log.md [hunk 1]
```diff
@@ -1,3 +1,28 @@
+# v 0.8.0 (2/12/2022)
+
+## Highlights
+- Unify the parsing method of config scripts, and support both local and pai platform products ([#235](https://github.com/alibaba/EasyCV/pull/235))
```
Comment (Cathy0908) [inline docs/source/change_log.md:4]: 这个移动到Improvements里

File: README.md [hunk 2]
```diff
@@ -46,8 +46,13 @@ EasyCV is an all-in-one computer vision toolbox based on PyTorch, mainly focuses
 
 
 ## What's New
+[🔥 2022.12.02]
```
Comment (wenmengzhou) [inline README.md:49]: 技术文档也更新一下

File: docs/source/change_log.md [hunk 3]
```diff
@@ -1,3 +1,28 @@
...
+
+## New Features
+- Add DeiT of Hydra Attention version ([#220](https://github.com/alibaba/EasyCV/pull/220))
+- Add EdgeViTv3 ([#214](https://github.com/alibaba/EasyCV/pull/214))
+- Add BEVFormer and improve the performance of BEVFormer ([#224](https://github.com/alibaba/EasyCV/pull/224))
+- Unify the parsing method of config scripts, and support both local and pai platform products ([#235](https://github.com/alibaba/EasyCV/pull/235))
+- Add DINO++ and support objects365 pretrain ([#242](https://github.com/alibaba/EasyCV/pull/242))
+
+## Improvements
+- Add data source and support for automatic download ([#206](https://github.com/alibaba/EasyCV/pull/206) [#229](https://github.com/alibaba/EasyCV/pull/229))
```
Comment (Cathy0908) [inline docs/source/change_log.md:16]: 加了哪些支持自动下载的数据集？

File: docs/source/change_log.md [hunk 4]
```diff
@@ -1,3 +1,28 @@
...
+- Classification support confusion matrix ([#241](https://github.com/alibaba/EasyCV/pull/241))
```
Comment (Cathy0908) [inline docs/source/change_log.md:17]: Add confusion matrix metric for Classification models

File: docs/source/change_log.md [hunk 5]
```diff
@@ -1,3 +1,28 @@
...
+
+## Bug Fixes
+- Update flake8 url for pre-commit ([#234](https://github.com/alibaba/EasyCV/pull/234))
```
Comment (Cathy0908) [inline docs/source/change_log.md:20]: 这个代码格式检查的可以删除

File: docs/source/change_log.md [hunk 6]
```diff
@@ -1,3 +1,28 @@
...
+- Sync the predict config in the config file for predictor ([#238](https://github.com/alibaba/EasyCV/pull/238))
+- Modify ocr predictor ([#237](https://github.com/alibaba/EasyCV/pull/237))
```
Comment (Cathy0908) [inline docs/source/change_log.md:22]: 修改了哪里，修复了什么bug？

File: docs/source/change_log.md [hunk 7]
```diff
@@ -1,3 +1,28 @@
...
+- MMMosaic bug fix ([#231](https://github.com/alibaba/EasyCV/pull/231))
```
Comment (Cathy0908) [inline docs/source/change_log.md:23]: 修复了什么bug

File: docs/source/change_log.md (hunk 3 above)
Comment (tuofeilunhifi) [reply docs/source/change_log.md:16]: https://yuque.antfin.com/uage03/qr8hul/dsoz3x

File: docs/source/change_log.md (hunk 1 above)
Comment (tuofeilunhifi) [reply docs/source/change_log.md:4]: done

File: README.md (hunk 2 above)
Comment (tuofeilunhifi) [reply README.md:49]: done

File: docs/source/change_log.md (hunk 4 above)
Comment (tuofeilunhifi) [reply docs/source/change_log.md]: done

File: docs/source/change_log.md (hunk 5 above)
Comment (tuofeilunhifi) [reply docs/source/change_log.md]: done

File: docs/source/change_log.md (hunk 6 above)
Comment (tuofeilunhifi) [reply docs/source/change_log.md]: done

File: docs/source/change_log.md (hunk 7 above)
Comment (tuofeilunhifi) [reply docs/source/change_log.md]: done

File: docs/source/change_log.md (hunk 3 above)
Comment (tuofeilunhifi) [reply docs/source/change_log.md]: done

Comment (tuofeilunhifi) [reply docs/source/change_log.md]: 放在 https://github.com/alibaba/EasyCV/blob/master/docs/source/data_hub.md 里面，加了跳转的链接

**Review by wenmengzhou — APPROVED** [review]

File: docs/source/change_log.md [hunk 8]
```diff
@@ -1,3 +1,25 @@
...
+
+## New Features
+- Add DeiT of Hydra Attention version ([#220](https://github.com/alibaba/EasyCV/pull/220))
+- Add EdgeViTv3 ([#214](https://github.com/alibaba/EasyCV/pull/214))
+- Add BEVFormer and improve the performance of BEVFormer ([#224](https://github.com/alibaba/EasyCV/pull/224))
+- Add DINO++ and support objects365 pretrain ([#242](https://github.com/alibaba/EasyCV/pull/242))
+
+## Improvements
+- Unify the parsing method of config scripts, and support both local and pai platform products ([#235](https://github.com/alibaba/EasyCV/pull/235))
+- Add data source and support for automatic download ([#206](https://github.com/alibaba/EasyCV/pull/206) [#229](https://github.com/alibaba/EasyCV/pull/229) [data_hub](https://github.com/alibaba/EasyCV/blob/master/docs/source/data_hub.md))
```
Comment (Cathy0908) [inline docs/source/change_log.md:15]: Add more data source apis for open source datasets, involving classification, detection, segmentation and keypoints tasks. And part of the data source apis support automatic download. For more information, please refer to [data_hub](https://github.com/alibaba/EasyCV/blob/master/docs/source/data_hub.md) ([#206](https://github.com/alibaba/EasyCV/pull/206) [#229](https://github.com/alibaba/EasyCV/pull/229))

**Pull request merged**